test/integration/buildruns_to_taskruns_test.go

```diff
@@ -355,6 +355,7 @@ var _ = Describe("Integration tests BuildRuns and TaskRuns", func() {
 			Expect(err).To(BeNil())
 
 			err = tb.GetBRTillDesiredReason(buildRunObject.Name, fmt.Sprintf("TaskRun \"%s\" was cancelled", tr.Name))
+			Expect(err).To(BeNil())
```

Comment:
This is a correct change. Though, I think it uncovers that `GetBRTillDesiredReason` should maybe also return the `actualReason` like `GetTRTillDesiredReason` so that the `Expect` can get a message like below. Do you want to change it as well or should this be done separately?

Reply:
Let try to understand your point and think about it.

Reply:
Since it touched anyway, you are right, we should synchronize it with the other function signature.

Reply:
Done